addons/account/tests/test_account_all_l10n.py

```diff
@@ -13,6 +15,17 @@ def test_all_l10n(env):
     As the module install is not yet fully transactional, the modules will
     remain installed after the test.
     """
+    def try_loading_patch(self, template_code, company, install_demo=True):
+        if not company:
+            company = self.env.company
+        if isinstance(company, int):
+            company = self.env['res.company'].browse([company])
+
+        template_code = template_code or company and self._guess_chart_template(company.country_id)
+        return self.with_context(l10n_check_fields_complete=True)._load(template_code, company, install_demo)
```

Comment:
Ok, but then something like this is better (didn't test but you get the idea; not sure about the `type`)
```suggestion
    try_loading = type(self.env['chart.template']).try_loading
    def try_loading_patch(self, template_code, company, install_demo=True):
        self = self.with_context(l10n_check_fields_complete=True)
        return try_loading(self, template_code, company, install_demo)
```